sys/include/net/gnrc/lwmac/types.h

```diff
@@ -204,7 +196,7 @@ typedef struct lwmac {
     uint8_t lwmac_info;                                         /**< LWMAC's internal informations (flags) */
     gnrc_lwmac_timeout_t timeouts[GNRC_LWMAC_TIMEOUT_COUNT];    /**< Store timeouts used for protocol */
 
-#if (GNRC_LWMAC_ENABLE_DUTYCYLE_RECORD == 1)
+#if (GNRC_MAC_ENABLE_DUTYCYCLE_RECORD == 1)
```

Comment:
I don't know if this is the *only* macro used here from "gnrc_mac", but it looks inconsistent regarding the rest of the macros namespaced as "GNRC_LWMAC". Maybe all the variables need to be repalced by GNRC_MAC ones? If so, it can be done in a follow-up PR.

Reply:
Hi, Paco, actually, the GNRC_MAC_ENABLE_DUTYCYCLE_RECORD I introduced here belongs to the gnrc_mac module, while the rest of the "GNRC_LWMAC" in this file (and other files of LWMAC) belongs to the gnrc_lwmac module.
So, there is no need to replace "GNRC_LWMAC" by GNRC_MAC ones. :-)

Reply:
Sounds OK, and since @miri64 already reviewed I'll just approve the changes :-) Thanks @zhuoshuguo !